### ISSUE TITLE
build fails in apps without config.css

### ISSUE THREAD

**Issue by tyrjo** [RallyApps/rally-app-builder#82]
It appears the app builder assumes config.css, however this is NOT set in many of the apps currently in the [app-catalog](https://github.com/RallyApps/app-catalog)

Trying to build `app-catalog/src/apps/releaseplanning` fails with the following error:

```
Compiling the App.
/Users/johty01/Development/RallyApps/app-catalog/node_modules/rally-app-builder/node_modules/async/lib/async.js:81
        if (!arr.length) {

TypeError: Cannot read property 'length' of undefined
    at Object.async.forEach (/Users/johty01/Development/RallyApps/app-catalog/node_modules/rally-app-builder/node_modules/async/lib/async.js:81:17)
[ snip ]
```

Building `app-catalog/src/apps/releasesummary` has no issues.

Adding the following to the releaseplanning `config.json` file fixes the problem

```
"css": []
```

**Comment by krmorse** [comment]
Yeah, I think the css block is a required element in config.json currently.  Did you edit your config.json in the app you working on to not have it?  I know rally app builder will generate the "css": [] by default.  I agree we could probably make it handle this a bit more gracefully.
